grails-app/domain/com/causecode/fileuploader/UFile.groovy

```diff
@@ -103,8 +105,8 @@ class UFile implements Serializable {
         new File(path).exists()
     }
 
-    String getContainer() {
-        containerName(Holders.flatConfig["fileuploader.groups.${fileGroup}.container"])
```

Comment:
Need to change the method name to fix the codeclimate issue of `Code readability confusion`  as fieldName `containerName` is a good fit here.